src/w3c/headers.js

```diff
@@ -395,18 +395,22 @@ export function run(conf) {
       }
     }
   };
+  if (!conf.formerEditors) conf.formerEditors = [];
   if (conf.editors) {
     conf.editors.forEach(peopCheck);
+    conf.formerEditors = conf.editors.reduce((collector, person) => {
```

Comment:
Probably want a comment here about what this is actually doing... relocating editor to former editor, iff `retiredDate` member is present.